Doxyfile

```diff
@@ -1,2565 +1,2565 @@
-# Doxyfile 1.8.15

```

Comment:
Need to verify that Doxygen is fine with just LF

Reply:
I tried to build doxygen with the required dependencies (clang I think) but I was unable to... Though, I looked in the doc and didn't find any mention of anything related to line endings

Reply:
Just tested, Doxygen is fine.